shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ShortCircuitAnalysis.java

```diff
@@ -66,6 +69,9 @@ public CompletableFuture<ShortCircuitAnalysisResult> runAsync(Network network,
             Objects.requireNonNull(faults, NOT_NULL_FAULT_MESSAGE);
             Objects.requireNonNull(computationManager, NOT_NULL_COMPUTATION_MANAGER_MESSAGE);
             Objects.requireNonNull(parameters, NOT_NULL_PARAMETERS_MESSAGE);
+            if (parameters.getInitialVoltageProfileMode() == InitialVoltageProfileMode.CONFIGURED && (parameters.getVoltageRangeData() == null || parameters.getVoltageRangeData().isEmpty())) {
+                throw new PowsyblException("Configured initial voltage profile but nominal voltage ranges with associated coefficients are missing.");
+            }
```

Comment:
This check is done at several places (in the previous `runAsync(...)` method, in `ShortCircuitParameters.load(...)` and in `ShortCircuitParametersDeserializer.load(...)`). It could be extracted to be shared (for instance in a new `ShortCircuitParameter.validate()` method).